.github/workflows/linting.yml

```diff
@@ -138,3 +78,5 @@ jobs:
             lint_log.txt
             lint_results.md
             PR_number.txt
+
+#
```

Comment:
?

Reply:
Template thats why I left it in 🤷 I thought there was a PR to remove trailing #, need to investigate